apps/state_mediator/test/state_mediator/integration/gtfs_test.exs

```diff
@@ -195,17 +195,31 @@ defmodule StateMediator.Integration.GtfsTest do
       assert [%{name: "Braintree - Alewife"}, %{name: "Ashmont - Alewife"}] = shapes_1
     end
 
-    test "Providence/Stoughton has 2 non-ignored shapes each direction" do
+    test "Providence/Stoughton has at least 2 non-ignored shapes in direction 0 and 2 in direction 1" do
       [shapes_0, shapes_1] = shapes_in_both_directions("CR-Providence")
 
       assert [
-               shape_0,
-               %{id: "9890004"},
-               %{id: "SouthStationToStoughtonViaFairmount"}
+               # South Station - Wickford Junction via Back Bay
+               shape_0
+               | remaining_shapes
              ] = shapes_0
 
       assert shape_0.name =~ "South Station - Wickford Junction"
 
+      assert Enum.all?(remaining_shapes, fn shape ->
+               case shape do
+                 # South Station - Stoughton via Back Bay
+                 %{id: "9890004"} -> true
+                 # South Station - Stoughton via Fairmount
+                 %{id: "SouthStationToStoughtonViaFairmount"} -> true
+                 # South Station - Wickford Junction
+                 %{id: "9890009"} -> true
+                 _ -> false
+               end
+             end)
```

Comment:
How about
```suggestion
      Enum.each(remaining_shapes, fn shape ->
        assert shape[:id] in ["9890004", "SouthStationToStoughtonViaFairmount", "9890009"]
      end)
```
So that if there's an unexpected shape in the list, the test shows which one it is?

(`shape[:id]` can be `shape.id` if all elements are maps with an `:id` field--I wasn't sure if that was the case)